util/fibers/submit_entry.h

```diff
@@ -130,6 +131,22 @@ class SubmitEntry {
     sqe_->off = offset;
   }
 
+  void PrepFSync(int fd) {
+    PrepFd(IORING_OP_FSYNC, fd);
+    // TODO fsync with flags = fdatasync
+    sqe_->sync_range_flags = 0;
+  }
+
+  void PrepStatX(std::string_view filepath, struct statx *stat) {
+    //AT_FDCWD is ignored when addr is an absolute path
+	  PrepFd(IORING_OP_STATX, AT_FDCWD);
```

Comment:
indentation

Reply:
<img width="745" height="248" alt="Screenshot 2025-09-24 at 10 00 29 AM" src="https://github.com/user-attachments/assets/ae8fc5eb-4cb3-4d2e-9767-a4434146d8d9" />


It's a github bug! I explicitly set `\t` to `space` in my vim so there are `no tab characters`